### PR TITLE
Add "float: right|left" support for css.

### LINKED ISSUE #1207: "align" attribute on images doesn't work.
### Describe the bug

The `align` attribute on images are being removed by the XSS plugin.

### Steps to reproduce

Paste the following example into a markdown field:

```md

<div>
<img align="right" width="50%" src="https://cdn.modrinth.com/data/3s19I5jr/images/2146f756a3c5ef7e916e8f3d02e36a1bfa8d19ad.png" />

## The Carousel Screen

The carousel is where all your skins will be stored for future use! You can create, edit and delete your presets.
  
You can customize every preset to your own desire in the edit screen.
<br><br><br><br><br><br>
</div>

<div>
<img align="left" width="50%" src="https://cdn.modrinth.com/data/3s19I5jr/images/2146f756a3c5ef7e916e8f3d02e36a1bfa8d19ad.png" />

## The Edit Screen

The edit screen allows you to change the actual texture of the skin, the model type and the name of the preset.

For the future, there are plans to add cape and ear customization to this screen.

</div>
```

### Expected behavior

The expected visualization is:

image

<div>

## The Carousel Screen

The carousel is where all your skins will be stored for future use! You can create, edit and delete your presets.
  
You can customize every preset to your own desire in the edit screen.
<br><br><br><br><br><br>
</div>

<div>

## The Edit Screen

The edit screen allows you to change the actual texture of the skin, the model type and the name of the preset.

For the future, there are plans to add cape and ear customization to this screen.

</div>

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by IMB11**
Would be good if this could get into production asap? A few mods need it from a brief search on discord

### CODE REVIEW

**Review by triphora — COMMENTED** [review]
Center?

**Comment by mineblock11** [comment]
Float does not have a "center" value.

https://developer.mozilla.org/en-US/docs/Web/CSS/float#values

File: helpers/parse.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const configuredXss = new xss.FilterXSS({
     whiteList: {
       'image-rendering': /^pixelated$/,
       'text-align': /^center|left|right$/,
+      'float': /^left|right$/,
```
Comment (brawaru) [inline helpers/parse.js:28]:
```suggestion
      float: /^left|right$/,
```

**Review by brawaru — CHANGES_REQUESTED** [review]
Maybe we should also have `align` attribute for elements which would allow for compatibility with GitHub READMEs? I believe this is just a matter of whitelisting the attribute and browsers already take care of it on their own.

**Comment by ThatGravyBoat** [comment]
if the `float` is added then `clear` should also be added to make it more compatible

**Comment by Prospector** [comment]
slightly confused, the linked issue is about align but this PR is about float?

**Comment by ThatGravyBoat** [comment]
I assume why this does the `float` CSS property instead of adding the `align` attribute is that the `align` attribute is deprecated while the `float` CSS property is not and is the recommended replacement for `align` for images.

**Comment by brawaru** [comment]
It being deprecated doesn't mean much, it's well supported by the browsers, and if you want to be sure, you can just re-implement it using simple CSS. But allowing it would ensure compatibility with Markdown from GitHub and other platforms.

<details>
<summary>SCSS</summary>

```scss
:not(img) {
  &[align="left"] {
    text-align: left;
  }
  
  &[align="middle"], &[align="center"] {
    text-align: center;
  }
  
  &[align="right"] {
    text-align: right;
  }
}

img {
  &[align="left"] {
    float: left;
    vertical-align: top;
  }
  
  &[align="right"] {
    float: right;
    vertical-align: top;
  }
  
  &[align="top"], &[valign="top"] {
    vertical-align: top;
  }
  
  &[align="middle"], &[valign="middle"] {
    vertical-align: middle;
  }
  
  &[align="bottom"], &[valign="bottom"] {
    vertical-align: bottom;
  }
}
```

(Can be wrapped in `.markdown-body { ... }` so that it only applies to Markdown)

</details>

**Review by triphora — APPROVED** [review]
Removed the linked issue since that's for something different.

**Pull request merged**